modules/local/tdf2mzml/main.nf

```diff
@@ -0,0 +1,38 @@
+process TDF2MZML {
+    tag "$meta.id"
+
+    container "docker.io/mfreitas/tdf2mzml"
```

Comment:
No singularity or conda?

Reply:
Unfortunately the tool is only available via dockerhub. Nextflow automatically converts the docker Container to singularity, but Conda is not supported for this Tool. It is only run for a specific input datatype though. There is no better alternative Tool atm.